python/meson.build

```diff
@@ -1,3 +1,3 @@
 py = import('python').find_installation()
 
-py.install_sources('lc3.py')
+py.install_sources('lc3py.py')
```

Comment:
Note that a PyPI project name does not have to be related to the module name. For example, "meson-python" is the name of the build backend, and its module import is "mesonpy".

Meson is "meson" on PyPI, but imports itself as "mesonbuild".

You could call it "lc3py" on PyPI but still have it import as "lc3", and IMHO it probably makes more sense that way considering you instead ended up doing "import lc3py as lc3", clearly it is more memorable to refer to the module as "lc3"...